lib/models/user_dictionary.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'user_dictionary.g.dart';
+
+@JsonSerializable()
+class UserDictionary {
+  @JsonKey(name: 'rk')
```

Comment:
Suggestion: Add comments here that explain what those values are

Reply:
I don't know yet ^^